qbraid/api/job_api.py

```diff
@@ -81,14 +81,20 @@ def init_job(
         "vendorJobId": vendor_job_id,
         "qbraidDeviceId": device.id,
         "vendorDeviceId": device.vendor_device_id,
-        "circuitNumQubits": circuit.num_qubits,
-        "circuitDepth": circuit.depth,
+        "circuitNumQubits": circuits[0].num_qubits if len(circuits) > 1 else -1,
```

Comment:
This change is actually breaking [`test_get_jobs_no_results()`](https://github.com/qBraid/qBraid/blob/33728764c03c3ffd18b7eebb8f4e94ffcc32d2ff/qbraid/tests/test_top_level.py#LL114C5-L114C29). Instead of a -1, we can just omit the field or put None.

Reply:
Or, we could even omit the field, and update the other test.